cmake/app/boilerplate.cmake

```diff
@@ -84,6 +84,12 @@ set(AUTOCONF_H ${__build_dir}/include/generated/autoconf.h)
 # Re-configure (Re-execute all CMakeLists.txt code) when autoconf.h changes
 set_property(DIRECTORY APPEND PROPERTY CMAKE_CONFIGURE_DEPENDS ${AUTOCONF_H})
 
+# Load additional default values from the user
+if (EXISTS ${ZEPHYR_BASE}/defaults.cmake)
+  message("Loading additional defaults")
```

Comment:
One last thing: shouldn't this be `message(STATUS "....")`?

If you don't specify STATUS, output goes to stderr instead of stdout:

https://cmake.org/cmake/help/v3.8/command/message.html

Is that what you meant to do?

Reply:
oops, thanks for pointing that out! will fix

Reply:
Fixed